app/src/main/java/io/islnd/android/islnd/app/adapters/PostAdapter.java

```diff
@@ -58,7 +59,7 @@ public void onBindViewHolder(GlancePostViewHolder holder, Cursor cursor) {
         );
 
         holder.postUserName.setText(post.getUserName());
-        holder.postTimestamp.setText(Util.smartTimestampFromUnixTime(post.getTimestamp()));
+        holder.postTimestamp.setText(Util.smartTimestampFromUnixTime(ServerTime.toLocalTimeMillis(post.getTimestamp())));
```

Comment:
This was kind of a code smell for me. I don't like that we are having to convert between server time and local time. Instead of converting something from server back to local, it might be simpler if we have the method that builds the smart timestamp use the server time when it does the difference. I'm not really sure what would be better here. I think changing that method is worth considering because that's the only usage of the `toLocalTimeMillis` method.